src/components/icons/doit.sh

```
@@ -0,0 +1,65 @@
+map["xx14"]="Falconers"
+map["xx07"]="bandage"
+map["xx22"]="Order"
+map["xx06"]="skull"
+map["xx31"]="Logo"
+map["xx15"]="Farmers"
+map["xx23"]="Order-GBCP"
+map["xx32"]="knock-down"
+map["xx11"]="Butchers"
+
+
+for file in ${!map[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)